docs/.vuepress/plugins/dump-docs-data/index.js

```diff
@@ -49,10 +50,12 @@ module.exports = (options, context) => {
 
       const docsVersions = await fetchDocsVersions();
       const canonicalURLs = await generateCommonCanonicalURLs(rawCanonicalURLs);
+      const minorsLimit = docsVersions.versions[0] === 'next' ? 7 : 6;
```

Comment:
The versions limit list is already applied in the `docs-version.js` file. Can we stay with the idea to have defined the limitation only in that file?

Reply:
Changed within https://github.com/handsontable/handsontable/pull/10079/commits/e539c2a3a296ae4735d921341bed747ab79092d7.